stages/02_process_embl.py

```diff
@@ -0,0 +1,11 @@
+from Bio import SeqIO
+import pandas as pd
+
+new_embl_records = SeqIO.parse("./download/miRNA.dat", "embl")
+# prev_embl_records = SeqIO.parse("./download/miRNA.dead", "embl")
```

Comment:
I figured out why the file could not be read, but we don't need it for now: the file has the `AC` field before the `ID` field. BioPerl's `embl` parser also fails on this, but its `embldriver` parser works.